content/addons/60fps.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Higher project framerate mode
+id: 60fps
+---
+
+**Higher project framerate mode** is an addon that allows customization of the project's run speed to be faster than the default of 30 FPS. This has the effect of making the project appear to run smoother and at a faster rate relative to the custom FPS value; therefore, the default value of 60 FPS will essentially make the project run twice as fast.
+
+The feature can be quickly toggled on/off by holding `alt` and clicking the green flag. The flag will turn blue and a yellow fast-forward icon will appear over it, indicating that the project is running at a faster speed.
+
+## Features
+
+- The addon's functionality is only enabled whenever the user toggles it on by holding `alt` and clicking the green flag. The addon's functionality is toggled off every time the page is opened/refreshed.
+- The addon works in both the project page and the editor.
+- By default, the addon (when toggled on) sets the project's framerate to 60 FPS. This value can be changed in the addon's settings to an integer ranging from 31 to 240.
+- To prevent abuse and cheating in cloud-based multiplayer games, the addon does not work whenever there are cloud variables in a shared project.
```

Comment:
Really?

Reply:
I don't think we check whether there are cloud variables or not.

Reply:
> @Hexagonl Was this removed (or the statement checked?)

It hasn't been removed. The statement is definitely incorrect - there are no references to cloud variables in the addon's code.

Reply:
```suggestion
```

Reply:
> @Hexagonl Was this removed (or the statement checked?)

I verified it myself and I did remove it in my latest commit